daemon/graphdriver/driver.go

```diff
@@ -35,6 +35,10 @@ var (
 	ErrPrerequisites = errors.New("prerequisites for driver not satisfied (wrong filesystem?)")
 	// ErrIncompatibleFS returned when file system is not supported.
 	ErrIncompatibleFS = fmt.Errorf("backing file system is unsupported for this graph driver")
+
+	// Optional, compile-time defined comma-separated list of
+	// preferred graph drivers, overriding the default `priority` list.
+	priorityOverride string
```

Comment:
This can (and should) be a `const`.

Reply:
It can not be a `const` because `const` can not be overridden with `-X`.